client/llb/source.go

```diff
@@ -316,6 +305,53 @@ func Git(remote, ref string, opts ...GitOption) State {
 	return NewState(source.Output())
 }
 
+func parseGitRemote(remote string) (int, string, string) {
+	var protocolType int
+	remote, protocolType = gitutil.ParseProtocol(remote)
+
+	if protocolType != gitutil.SSHProtocol {
+		return protocolType, "", remote
+	}
+	sshHost := remote
+	path := ""
+	parts := strings.SplitN(remote, "/", 2)
+	if len(parts) == 2 {
+		sshHost = parts[0]
+		path = parts[1]
+	}
+
+	hostParts := strings.SplitN(sshHost, ":", 2)
+	if len(hostParts) >= 2 {
+		// check for a possible port number after the host
+		if !isNumeric(hostParts[1]) {
+			sshHost = hostParts[0]
+			pathPrefix := strings.Join(hostParts[1:], "/")
```

Comment:
This seems to be no-op as `hostParts[1:]` always only has one item because of the `SplitN`

Reply:
This was left over from when it handled multiple `:`s. Removed.